src/ducks/programs.js

```diff
@@ -35,10 +35,10 @@ const i2a = {
 };
 
 const darian = {
```

Comment:
Dangit Darian, go home and bring back your brother Darién!

Reply:
hahah you have no idea how many time I kept getting it wrong while working on this.